### DISCUSSION TITLE
Welcome to ansible-lint Discussions!

### DISCUSSION THREAD

**Discussion by gundalow** [ansible/ansible-lint#1047] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ssbarnea** [comment]
Thanks for doing this. I hope this will prove to be a better communication channel for user support and other discussions around the linter.

This should also help us avoid support tickets from the issue tracker,  making easier to track only confirmed bugs or features.
